admin/apple-actions/index/class-export.php

```diff
@@ -49,6 +49,9 @@ public function perform() {
 	 * @access public
 	 */
 	public function fetch_exporter() {
+
+		do_action( 'apple_news_before_generate_article', $this->id );
```

Comment:
I think the filter name is misleading since it's not in `generate_article`. I'd prefer this is called `apple_news_do_fetch_exporter`.